### DISCUSSION TITLE
Move version comment to a plugin of its own to lose the preFlight dependency

### DISCUSSION THREAD

**Discussion by willdean** [tailwindlabs/tailwindcss#9207] [Ideas]
At the moment, you only get a version comment at the top of the TW output if you're using preFlight.
To me, the use (or not) of the preFlight and the value of a version comment are orthogonal - the former is sometimes useful, sometimes not.  The latter is always a nice thing to have, and I miss it on non-preFlighted projects.

I tentatively propose adding a new plugin "versionComment" at the top of the corePlugins object, something like this:
(I am not God's Gift to TW plugin authorship so this may be embarrassingly bad)

```   
 versionComment: ({ addBase  })=>{
        addBase([
            _postcss.default.comment({
                text: `! tailwindcss v${_packageJson.version} | MIT License | https://tailwindcss.com`
            })
        ]);
    },
```

And then removing the existing code which generates the comment from the preFlight plugin.

One also has to add `"versionComment"` at the top of the list of core plugins in corePluginList.js

Is this something which might be considered if I submitted a PR?

**Comment by MichaelAllenWarner** [comment]
Not sure if a separate plugin is the right solution, but I'm bumping this because I agree that the license-comment should appear even if you have preflight disabled.

Here is a Tailwind Play link with preflight disabled, and you can see that there's no license-comment at the top of the Generated CSS: https://play.tailwindcss.com/nPTMCxCuB5?file=config
